js/editor.js

```
@@ -674,6 +961,16 @@ wisk.editor.htmlToMarkdown = function (html) {
                 return `~~${result}~~`;
             case 'u':
                 return `__${result}__`;
+            case 'code':
+                return `\`${result}\``;
+            case 'sup':
+                // Check if it contains a reference number link
+                const supRefLink = node.querySelector('a.reference-number');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)